docs/draft/9-3rd-party.md

```diff
@@ -0,0 +1,122 @@
+---
+permalink: /spec/9
+parent: Stable specs
+title: 9/3RD-PARTY
+---
+
+# 9/3RD-PARTY
+
+> Version: 0.1
+>
+> Status: Draft
```

Comment:
Is it draft or stable? Above indicates stable

Reply:
Changed to draft